CodeReady Containers/DaemonCommander/Handlers.swift

```diff
@@ -193,7 +193,7 @@ func HandleStart(pullSecretPath: String) {
         } else {
             response = try SendCommandToDaemon(command: Request(command: "start", args: ["pullSecretFile": pullSecretPath]))
         }
-        displayNotification(title: "CodeReady Containers", body: "Starting OpenShift Cluster, this could take a few minutes..")
+        displayNotification(title: "Starting Cluster", body: "Starting OpenShift Cluster, this could take a few minutes..")
```

Comment:
With unified form I meant tittle referring to cluster or Codeready Containers apart from adding the action. So now we got:
starting notification: tittle refers to Ccluster, body to Openshift Cluster
started notification: tittle refers to CodeReady Containers, body to Openshift Cluster
stopped notification: tittle refers to Cluster, body to CRC Cluster 

I think the messages should refer to the same thing

Reply:
Removed all occurrences of "CodeReady Containers" from the title, and the body only talks about OpenShift cluster, i removed CodeReady as the icon and the main title makes it clear the notification is from CRC!